conceptual/EFCore.PG/mapping/enum.md

```diff
@@ -75,7 +75,7 @@ using (var ctx = new MyDbContext())
 
 ## Altering enum definitions
 
-Although PostgreSQL allows [altering enum types](https://www.postgresql.org/docs/current/static/sql-altertype.html), the Npgsql provider currently does not generate SQL for those operations (beyond creating and dropping the entire type). If you to add or rename enum values (removal is not supported by PostgreSQL), you'll have to include raw SQL in your migrations (this is quite easy to do). As always, test your migrations carefully before running them on production databases.
+While PostgreSQL allows [altering enum types](https://www.postgresql.org/docs/current/static/sql-altertype.html), the Npgsql provider currently only [supports adding new values](https://github.com/npgsql/efcore.pg/pull/1183) (beyond creating and dropping the entire type). If you to want to rename enum values (removal is not supported by PostgreSQL), you'll have to include raw SQL in your migrations (this is quite easy to do). As always, test your migrations carefully before running them on production databases.
```

Comment:
```suggestion
The Npgsql provider only allow adding new values to existing enums: PostgreSQL itself doesn't support removing enum values, and while renaming is supporting, it isn't automatically done by the provider to avoid unreliable heuristics for detecting the renaming. Renaming an enum value can be done by including [raw SQL](https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/managing?tabs=dotnet-core-cli#arbitrary-changes-via-raw-sql) in your migrations as follows:

```c#
migrationBuilder.Sql(@"ALTER TYPE mood RENAME VALUE happy TO thrilled;");
```
  
As always, test your migrations carefully before running them on production databases.
```